### PR TITLE
Enable homepage as markdown file

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Enable homepage as markdown file

## QA

- Run ./run to get started
- Go to http://0.0.0.0:8015/
- Make sure the copy is the same as the copy doc and that there are feeds from insights and tutorials

Doc: https://docs.google.com/document/d/1bvQDL9UlLMx8DFNQTCELPB4I3qBNwSV-OAig7z5-Jts/edit

### CODE REVIEW

File: templates/includes/components/homepage.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+        <div class="p-divider">
+            <div class="p-strip">
+                <div class="u-equal-height">
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/includes/components/homepage.html]: Otherwise, you can't use more then 12-col in a strip.

File: templates/pages/index.html [hunk 2]
```diff
@@ -4,20 +4,22 @@
...
-        
-        <div class="row">
-            <div class="col-6">
-                <h2>Build an embedded device</h2>
-                <p>Use Ubuntu Core to secure and keep your IoT deployments up to date.</p>
-            </div>
-            <div class="col-6">
-                <h2>Package and distribute your code</h2>
-                <p>Use Snapcraft to publish apps to Linux desktop, server, cloud and devices.</p>
+    <main id="main-content" class="p-layout__inner">
```
Comment (anthonydillon) [inline templates/pages/index.html]: Is this file used?

File: templates/pages/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+----
+title: Ubuntu developer
+description: Snapcraft summary page
+page_type: modular
+insights_feed:
+    tag: developer
+tutorial_feed: true
+----
+
+# Millions of developers use Ubuntu to write, build and deploy their code
```
Comment (anthonydillon) [inline templates/pages/index.md:10]: Is this file used?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some questions in the code and QA was ok but missing content from "Target platforms" down.

File: templates/includes/components/homepage.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+        <div class="p-divider">
```
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/includes/components/homepage.html]: No, fixed

File: templates/includes/components/homepage.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/includes/components/homepage.html]: Added

File: templates/pages/index.html (hunk 2 above)
Comment (grahambancroft) [reply templates/pages/index.html]: it’s not, I’ve removed

File: templates/pages/index.md (hunk 3 above)
Comment (grahambancroft) [reply templates/pages/index.md:10]: it is yes

**Review by anthonydillon — APPROVED** [review]

**Comment by anthonydillon** [comment]
@grahambancroft Thanks :+1:

**Pull request merged**